Tasks02_05/task03.py

```diff
@@ -13,8 +13,27 @@
         ...
 
 """
+import math
 from typing import Tuple
 
 
 def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
-    ...
+    """
+    read a file line by line that consists of str
+    str include numbers (separated with commas)
+    :param file_name: name of file + .txt
+    :return: Tuple(min_value, max_value) for each line
+    """
+    with open(file_name) as fi:
+        for line in fi:
+            line = line.split(",")
+            min_num = math.inf
+            max_num = -math.inf
+            for i in line:
+                i = int(i)
+                if i <= min_num:
+                    min_num = i
+                if i >= max_num:
+                    max_num = i
+            result_tuple = (min_num, max_num)
+            yield result_tuple
```

Comment:
why is it yield here & why is it inside for-loop? function should return mix & max from all file, not from each line